{"uuid": "a3766568-bbee-426e-a89c-4ea9a00ae707", "counter": 7, "stdout": "\u001b[1;31mfatal: [192.168.1.1]: UNREACHABLE! => {\"changed\": false, \"msg\": \"Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused\", \"unreachable\": true}\u001b[0m", "start_line": 5, "end_line": 6, "runner_ident": "116baee4-3f0d-459a-b598-8d9c22ec90e5", "event": "runner_on_unreachable", "pid": 10352, "created": "2022-04-22T13:59:09.242561", "parent_uuid": "9e73ed77-7fb2-8246-02cc-00000000000f", "event_data": {"playbook": "deploy.yaml", "playbook_uuid": "3e0055f6-0d6e-46e4-906b-38f3722dd1ec", "play": "this is a  hello world example", "play_uuid": "9e73ed77-7fb2-8246-02cc-000000000007", "play_pattern": "all", "task": "Gathering Facts", "task_uuid": "9e73ed77-7fb2-8246-02cc-00000000000f", "task_action": "gather_facts", "task_args": "", "task_path": "/Users/devops/data/project/python/pgoops/upload/iac/helloworld/project/deploy.yaml:2", "host": "192.168.1.1", "remote_addr": "192.168.1.1", "start": "2022-04-22T13:59:09.222391", "end": "2022-04-22T13:59:09.242472", "duration": 0.020081, "res": {"unreachable": true, "msg": "Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused", "changed": false}, "uuid": "a3766568-bbee-426e-a89c-4ea9a00ae707"}}